src/prefect/deployments/runner.py

```diff
@@ -607,6 +643,7 @@ def from_entrypoint(
         work_pool_name: Optional[str] = None,
         work_queue_name: Optional[str] = None,
         job_variables: Optional[dict[str, Any]] = None,
+        sla: Optional[Union[SlaTypes, list[SlaTypes]]] = None,  # experimental
```

Comment:
Let's add an underscore to `sla` since this is a public method.
```suggestion
        _sla: Optional[Union[SlaTypes, list[SlaTypes]]] = None,  # experimental
```